### ISSUE TITLE
Temporal padding issue when temporal_patch_size != 2

### ISSUE THREAD

**Issue by JJJYmmm** [QwenLM/Qwen2.5-VL#1027]
The code snippets below do not work when temporal_patch_size != 2
1) https://github.com/huggingface/transformers/blob/064cd7cdaca7c9dc7d4359434164f2c2c17505d4/src/transformers/models/qwen2_vl/image_processing_qwen2_vl.py#L277-L279
2) https://github.com/huggingface/transformers/blob/064cd7cdaca7c9dc7d4359434164f2c2c17505d4/src/transformers/models/qwen2_vl/image_processing_qwen2_vl_fast.py#L228-L230

origin
```python
if patches.shape[0] % temporal_patch_size != 0:
    repeats = patches[-1].unsqueeze(0).repeat(temporal_patch_size - 1, 1, 1, 1)
    patches = torch.cat([patches, repeats], dim=0)
```

quick fix
```python
repeats = patches[-1].unsqueeze(0).repeat(temporal_path_size - patches.shape[0] % temporal_path_size, 1, 1, 1)
```
